### ISSUE TITLE
[Website] Add link to torchtime PyTorch code repository

### ISSUE THREAD

**Issue by philipdarke** [uea-machine-learning/tsml_repo#75]
I recently published [`torchtime`](https://github.com/philipdarke/torchtime), a Python package that downloads and prepares the TSC data sets as PyTorch tensors for deep learning research.

**It would be great to add a link to `torchtime` under TSC Code on the main website page.**

It sits on `sktime` and adds:

* control of training/validation/test splits
* missing data simulation
* options to impute missing data
* ability to append channels with time stamps, missing data masks and the time since last observation (as used by various deep learning TSC models)
* reproducible data processing by setting a random seed

Documentation is here [https://philipdarke.com/torchtime](https://philipdarke.com/torchtime).

Thanks for your continued work on the repository, it's such a great resource.

**Comment by TonyBagnall** [comment]
hi, happy to do that, will add it to my list for the next update

**Comment by TonyBagnall** [comment]
hi, we have added a link here, https://timeseriesclassification.com/ we will try it out after the summer, its looks really helpful because our first look at PyTorch left us thinking it was not as easy to do as tensorflow. 

Let me know if the text is ok, and if it is, could you close this issue? thanks. Tony

**Comment by philipdarke** [comment]
Looks good to me - you can link to the repository directly https://github.com/philipdarke/torchtime if preferred but up to you. Thanks again and do get in touch if you have any comments or issues with the package.
